### PR TITLE
Add installation guide

### PR DESCRIPTION
**Pull request by adenosie**

### CODE REVIEW

**Review by Riey — COMMENTED** [review]
Can you add newline under `###`?

File: README.md [hunk 1]
```diff
@@ -19,6 +19,45 @@ Korean IME
...
+## Configuration
+add the folowing to .xprofile or .xinitrc and restart X:
+
+```sh
+export GTK_IM_MODULE=kime
+kime-xim &
+export XMODIFIERS=@im=kime
+```
+
+read (CONFIGURATION.md) for detailed options.
```
Comment (Riey) [inline README.md:63]: need link for CONFIGURATION.md

File: README.md [hunk 2]
```diff
@@ -19,6 +19,45 @@ Korean IME
...
+
+sudo install -Dm755 kime-xim -t "/usr/bin"
+sudo install -Dm755 im-kime.so -t "/usr/lib/gtk-3.0/3.0.0/immodules"
+sudo install -Dm755 libkime_engine.so -t "/usr/lib"
+sudo install -Dm644 kime_engine.h -t "/usr/include/kime"
+sudo install -Dm644 config.yaml -t "/etc/kime"
+```
+
+## Configuration
+add the folowing to .xprofile or .xinitrc and restart X:
```
Comment (Riey) [inline README.md:55]: following

**Review by Riey — CHANGES_REQUESTED** [review]

**Comment by adenosie** [comment]
are the commands in 'Build from source' fine btw?
I just c/ped from aur pkgbuild

**Comment by Riey** [comment]
It's fine I'll add install.sh then update guide

**Comment by adenosie** [comment]
I think it's pretty done...

**Review by Riey — APPROVED** [review]

**Comment by Riey** [comment]
Looks good!

**Pull request merged**